lib/ansible/plugins/lookup/secret_server.py

```diff
@@ -0,0 +1,226 @@
+# Copyright (c) 2018 Michal Taratuta <michalta@Softcat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
```

Comment:
This file needs an `ANSIBLE_METADATA` indicating that it is community supported.